File: examples/examples/Python examples/gocodeo.py
def high_low(numbers):
    highest = numbers[0]
    lowest = numbers[0]
    
    for number in numbers:
        if number > highest:
            highest = number
        
        if number < lowest:
            lowest = number
    
    return (highest, lowest)


def is_palindrome(string):
    reversed_string = string[::-1]
    
    return string == reversed_string


def get_primes(num):
    primes = []
    
    for i in range(2, num + 1):
        is_prime = True
        
        for j in range(2, int(i ** 0.5) + 1):
            if i % j == 0:
                is_prime = False
                break
        
        if is_prime:
            primes.append(i)
    
    return primes   

File: examples/examples/Js examples/gocodeo.js

function highLow(numbers) {
    let highest = numbers[0];
    let lowest = numbers[0];
    
    for (let i = 1; i < numbers.length; i++) {
      if (numbers[i] > highest) {
        highest = numbers[i];
      }
      
      if (numbers[i] < lowest) {
        lowest = numbers[i];
      }
    }
    
    return { highest, lowest };
  }


  function getPrimes(num) {
    const primes = [];
    
    for (let i = 2; i <= num; i++) {
      let isPrime = true;
      
      for (let j = 2; j <= Math.sqrt(i); j++) {
        if (i % j === 0) {
          isPrime = false;
          break;
        }
      }
      
      if (isPrime) {
        primes.push(i);
      }
    }
    
    return primes;
  }


  function maxProfit(prices) {
    let minPrice = prices[0];
    let maxProfit = 0;

    for (let i = 1; i < prices.length; i++) {
        if (prices[i] < minPrice) {
            minPrice = prices[i];
        } else {
            maxProfit = Math.max(maxProfit, prices[i] - minPrice);
        }
    }
    return maxProfit;
   }